news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/good-night.jpg)](https://osu.ppy.sh/beatmapsets/679935)
+
+[Chaoslitz](https://osu.ppy.sh/users/3621552) and Moecho’s collab on colate and Nanahira’s "Good Night" is a great example of how collab difficulties are supposed to be mapped: the styles of both mappers blending with each other, resulting in a smooth playing experience without either of the mappers’ parts outshining the other. Coupled with the intuitive rhythm and patterns, the result is a very enjoyable map to play for players of all skillsets and levels.
+
+
+## <a name="taiko" id="taiko"></a>osu!taiko
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/passage.jpg)](http://osu.ppy.sh/beatmapsets/587426)
+
+The winning mapset of this season's beatmap Spotlight is nothing less than The Flashbulb's "Passage D", mapped by [Raiden](https://osu.ppy.sh/users/2239480)!
+
+A song known across several games and game modes, it has found its way into osu!taiko's ranked section a few months after being featured as a map in the 2018 osu!taiko World Cup Finals. Coincidence? Who knows. What I do know is that this entire map set is incredibly well-mapped, which is a big accomplishment given the calm yet chaotic nature of this song. 
+
+Raiden has managed to make an Ura Oni (named Raidcore, a difficulty name he uses for many of his high-difficulty maps) which balances playability and a huge amount of various snaps very well. It's not uncommon to find 4 different snaps in one chain of 1/4 notes, the common snapping. This is difficult to read, difficult to play, and difficult to map well. And yet, it follows the song very nicely, its flow (and "anti-flow") works, and that extends to all difficulties.
+
+A very well-deserved #1 spot for this season's Spotlight!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/sayonara.jpg)](http://osu.ppy.sh/beatmapsets/765430)
+
+Our runner-up mapset for this season, Chroma's "Sayonara Planet Wars" single-handedly mapped by [Greenshell](https://osu.ppy.sh/users/8693851) sure was a strong contender. Featuring one of the best (and nowadays, quite rare to find) lower difficulties in the sense that, while following the current criterias, it doesn't lose its identity at all, to the point you could recognize what song is playing behind this rather unique mapping.
+
+In the other hand, we got this really fun top diff that stands in here to tell us that, in a world ruled by techy 1/4+1/6 maps and high BPM speed maps, stream maps can still be quite fun and a worthy challenge even for the best of us.
```

Comment:
On the other hand only works if you used "on one hand" previously. Change to "On a side note / In other news / In addition"